frontend/Register/components/Button/index.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react';
```

Comment:
Do we really need styles button for checkout skeleton? Can we wait for React contexted components?

Reply:
Removed it, as there is a simpler way.